.eslintrc.js

```diff
@@ -111,6 +112,7 @@ module.exports = {
       plugins: ['@typescript-eslint'],
       rules: {
         '@typescript-eslint/consistent-type-imports': 'error',
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
I agree. We should avoid it where possible but sometimes libraries need to be permissive or ignorant about the types that are passed through APIs.